### PR TITLE
Reorg Paraobject Pages

### LINKED ISSUE #4099: Adding Parachain Implementer Guide Part 1
This Issue focuses on adding information contained in the [Parachain Implementer Guide](https://paritytech.github.io/polkadot/book/protocol-overview.html)

Content
- [ ] Why the need of Parachains
- [ ] Protocol Overview

### PR DESCRIPTION
**Pull request by filippoweb3**
See linked issue.

### CODE REVIEW

File: docs/learn/learn-parachains-faq.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+by whom. {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} guarantees valid state
+transitions. Executing a block finality outside the context of the relay chain is outside the scope
+of trust that {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} provides.
+
+### How about parachains that are not Substrate-based?
+
+Substrate provides [FRAME Pallets](https://docs.substrate.io/main-docs/fundamentals/runtime-intro/)
+as part of its framework to seamlessly build a rustic-based blockchain. Part of FRAME are pallets
+that can be used for consensus. {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }}
+being a Substrate-based chain relies on BABE as the block production scheme and GRANDPA as the
```
Comment (alfarok) [inline docs/learn/learn-parachains-faq.md]:
```suggestion
being a Substrate-based chain rely on BABE as the block production scheme and GRANDPA as the
```

File: docs/learn/learn-parachains-faq.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+be able to recognize a disruption as soon as it occurs and should stop producing block candidates.
+Likewise, it should be easy for them to recognize when it's safe to restart block production -
+perhaps based on finality delay, validator set size or some other factor that is yet to be decided
+within [Cumulus](https://github.com/paritytech/cumulus).
+
+### Parachain Development Kits (PDKs)
+
+Parachain Development Kits are a set of tools that enable developers to create their own
+applications as parachains. For more information, see the
+[PDK content](../build/build-parachains.md#parachain-development-kit-pdk). Please see the [Parachain Development page](../build/build-parachains.md) for more information.
```
Comment (alfarok) [inline docs/learn/learn-parachains-faq.md]:
```suggestion
PDK content](../build/build-parachains.md#parachain-development-kit-pdk) and [Parachain Development page](../build/build-parachains.md).
```

File: docs/learn/learn-parachains-faq.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+### Is security correlated to the number of validators? What about the number of parachains?
+
+Security is independent of the number of parachains that are connected to the Polkadot Relay Chain.
+The correlation of security and the number of validators exists as the higher number of validators
+will give the network stronger decentralization properties and make it harder to try to take down.
+However, the biggest indicator of the security of the network is the economic signal of the number
+of DOT that are bonded and staked. The greater the number of DOT staked by honest validators and
+nominators, the higher the minimum amount of DOT an attacker would need to acquire a validator slot.
+
+### Will parachains ever need their own security? In what scenarios do parachains need their own security?
```
Comment (alfarok) [inline docs/learn/learn-parachains-faq.md]:
```suggestion
### In what scenarios do parachains need their own security?
```

File: docs/learn/learn-parachains-faq.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+## Slot Auctions
+
+### How will parachain slots be distributed?
+
+Parachain slots are acquirable through auction. For more information on the auction process, please 
+see the {{ polkadot: [parachain slot auctions](learn-auction.md) :polkadot }}
+{{ kusama: [parachain slot auctions](learn-auction.md) :kusama }} article. Additionally, some parachain
+slots will be set aside to run {{ polkadot: [parathreads](learn-parathreads.md) :polkadot }}
+{{ kusama: [parathreads](learn-parathreads.md) :kusama }} &mdash; chains that bid on a per-block
```
Comment (alfarok) [inline docs/learn/learn-parachains-faq.md]:
```suggestion
see the [parachain slot auctions](learn-auction.md) article. Additionally, some parachain
slots will be set aside to run [parathreads](learn-parathreads.md) &mdash; chains that bid on a per-block
```
I know this was previously existing but conditional rendering isn't required here.

File: docs/learn/learn-parachains.md [hunk 5]
```diff
@@ -16,12 +16,14 @@ on the parachain development guide.
...
 
+## Definition of a Parachain
+
 A parachain is an application-specific data structure that is globally coherent and validatable by
 the validators of the Relay Chain. They take their name from the concept of parallelized chains that
 run parallel to the Relay Chain. Most commonly, a parachain will take the form of a blockchain, but
 there is no specific need for them to be actual blockchains.
 
-![One parachain](../assets/network/one_parachain.png)
+![One parachain](../assets/one-parachain.png)
```
Comment (alfarok) [inline docs/learn/learn-parachains.md:26]: Can the old `one_parachain.png` image be moved into `./assets/_legacy` or removed?

File: docs/learn/learn-parachains.md [hunk 6]
```diff
@@ -128,10 +193,10 @@ automatically become parathreads.
...
-"Common Good" parachains are parachain slots reserved for functionality that benefits the
-ecosystem as a whole. By allocating a subset of parachain slots to common good chains, the entire
-network can realize the benefit of valuable parachains that would otherwise be underfunded due to
-the free-rider problem. They are not allocated via the parachain auction process but by the on-chain
+"Common Good" parachains are parachain slots reserved for functionality that benefits the ecosystem
+as a whole. By allocating a subset of parachain slots to common good chains, the entire network can
+realize the benefit of valuable parachains that would otherwise be underfunded due to the free-rider
+problem. They are not allocated via the parachain auction process but by the on-chain
 {{ polkadot: [governance](learn-governance.md) :polkadot }}
 {{ kusama: [governance](learn-governance.md) :kusama }} system. Generally, a common good parachain's
```
Comment (alfarok) [inline docs/learn/learn-parachains.md]:
```suggestion
[governance](learn-governance.md) system. Generally, a common good parachain's
```

File: polkadot-wiki/sidebars.js [hunk 7]
```diff
@@ -130,7 +129,7 @@ module.exports = {
                 "learn/learn-common-goods",
                 "learn/learn-auction",
                 "learn/learn-crowdloans",
-                "learn/learn-teleport",
```
Comment (alfarok) [inline polkadot-wiki/sidebars.js:127]: @filippoweb3 did you intend to remove `learn/learn-teleport` also?  I just see the `learn/learn-security` doc removed  in this PR and replaced with `learn/learn-parachains-faq`

**Review by alfarok — COMMENTED** [review]
Hey @filippoweb3, the migration of content looks good to me!  I added a few suggestions around content even though I know the main purpose of this PR was organization of existing content.  I also have one question in regards to the `learn/learn-teleport` page.

I will give this another look tomorrow after you have a chance to reply and we can get it merged ASAP.

**Comment by alfarok** [comment]
I just noticed there is also a merge conflict in `polkadot-wiki/docusaurus.config.js`, likely from a change from another PR while you were working on this one.  Let me know if you have any questions or have issues getting it resolved.

File: docs/learn/learn-parachains.md (hunk 5 above)
Comment (filippoweb3) [reply docs/learn/learn-parachains.md:26]: moved to _legacy. Will do so for other figures in the future ;)

File: polkadot-wiki/sidebars.js (hunk 7 above)
Comment (filippoweb3) [reply polkadot-wiki/sidebars.js:127]: yes as the same page is present in Basics/Tokens and Assets. Not sure is a good thing to put the same page twice, we might want to either put it under parachains or under tokens and assets.

**Comment by filippoweb3** [comment]
Conflict was resolved, it was do to a redirect I added in a previous PR. I kept all redirects.

**Review by alfarok — APPROVED** [review]
Thanks for addressing the broken links!  LGTM

**Pull request merged**